### DISCUSSION TITLE
Static media failure when trying to log in

### DISCUSSION THREAD

**Discussion by cwt3c** [netbox-community/netbox#7831] [General]
Howdy!

I'm receiving this mesage when trying to log in:

> 
> **Static Media Failure**
> The following static media file failed to load: netbox-print.css
> 
> Check the following:
> 
> manage.py collectstatic was run during the most recent upgrade. This installs the most recent iteration of each static file into the static root path.
The HTTP service (e.g. nginx or Apache) is configured to serve files from the STATIC_ROOT path. Refer to the installation documentation for further guidance.
> 
> STATIC_URL: /static/
> The file netbox-print.css exists in the static root directory and is readable by the HTTP process.
> 
> Click here to attempt loading NetBox again.

When I reload the page/login URL everything works. 

I receive this error on 2 different new installations: Debian 11 and Ubuntu 20.04 LTS where I followed the installation guide from readthedocs.io step by step. During my installations no errors occured. 

Database type: Postgresql
Webserver: NGINX
Python3 version: 3.9.2
Netbox version: v3.0.9 & v.3.0.10 (tried both versions on clean systems)

Any hints on this? I already searched for a solution but found nothing suitable.

Thx in advance :-)

Chris

**Comment by candlerb** [comment]
Permissions error? #7511

**Comment by cwt3c** [comment]
Thx - that did the trick. 

The master process runs as root and not www-data. Setting the appropiate ownership helped.

**Reply by Sh4wn9** [reply]
@cwt3c Im having this issue, What permissions needed to be set to correct this problem?
